mobile/lib/features/brag/brag.dart

```diff
@@ -0,0 +1,426 @@
+import 'dart:io';
+
+import 'package:bitcoin_icons/bitcoin_icons.dart';
+import 'package:card_swiper/card_swiper.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get_10101/common/color.dart';
+import 'package:get_10101/common/domain/model.dart';
+import 'package:get_10101/features/brag/github_service.dart';
+import 'package:get_10101/features/trade/domain/direction.dart';
+import 'package:get_10101/features/trade/domain/leverage.dart';
+import 'package:get_10101/logger/logger.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:screenshot/screenshot.dart';
+import 'package:provider/provider.dart';
+import 'package:share_plus/share_plus.dart';
+
+class BragWidget extends StatefulWidget {
+  final String title;
+  final VoidCallback onClose;
+  final Direction direction;
+  final Leverage leverage;
+  final Amount? pnl;
+  final int? pnlPercent;
+  final Usd entryPrice;
+
+  const BragWidget(
+      {super.key,
+      required this.title,
+      required this.onClose,
+      required this.direction,
+      required this.leverage,
+      required this.pnl,
+      required this.entryPrice,
+      this.pnlPercent});
+
+  @override
+  State<BragWidget> createState() => _BragWidgetState();
+}
+
+class _BragWidgetState extends State<BragWidget> {
+  ScreenshotController screenShotController = ScreenshotController();
+  int selectedIndex = 0;
+  var images = [
+    "https://github.com/bonomat/memes/blob/main/images/laser_eyes_portrait.png?raw=true",
+    "https://github.com/bonomat/memes/blob/main/images/leoardo_cheers_portrait.png?raw=true",
+    "https://github.com/bonomat/memes/blob/main/images/do_something_portrait.png?raw=true",
+    "https://github.com/bonomat/memes/blob/main/images/got_some_sats_portrait.png?raw=true",
+    "https://github.com/bonomat/memes/blob/main/images/are_you_winning_son_always_have_been_portrait.png?raw=true"
```

Comment:
❓Why do we need to specify the memes here? I thought we would dynamically fetch them from the repository. Doesn't this mean that we have to make a code change if we want to add a new meme?

Reply:
Good catch, that's outdated and was just for testing.